js-pkg/apps/demos/src/pages/index.tsx

```diff
@@ -32,6 +32,11 @@ const demos = [
     title: 'WebRTC Chat',
     href: '/webrtc',
     description: 'Simple chat with WebRTC datachannels, using DriftDB for signaling.'
+  },
+  {
+    title: 'WebRTC presence',
+    href: '/webrtc-canvas',
+    description: 'cursors with ghosts!'
```

Comment:
Nice description haha but we should be clearer about exactly what we're trying to demonstrate with this demo and what that has to do with DriftDB.

Also, maybe `webrtc-presence` is a better href?

And capitalize both words in the title, like the other titles.